Automattic/wp-calypso
# SSH add screen: Redirect the user back to hosting configuration

#### Proposed Changes

If a user enabled SSH access for their site but doesn't yet have a SSH key on their account, they'll see this UI:


![image](https://user-images.githubusercontent.com/36432/198686994-39b70299-bd88-4cda-8b8a-f7b9e9f0ebf5.png)

If they were to click "Add an SSH key to your account", they end up on /me/security/ssh-key with no obvious way to return to Hosting Configuration.

However there is no clear way to navigate back to the previous page.

This pr fixes that by redirecting the user back to the ssh access screen after the user has
successfully saved his ssh key.


#### Testing Instructions

1. Navigate to hosting configuration screen under `Settings > Hosting Configuration`
2. Make sure your site has ssh access. Activate it if not. More info can be found [here](https://wordpress.com/support/connect-to-ssh-on-wordpress-com/#get-word-press-com-ssh-credentials-and-enable-ssh)
3. Press `Add an SSH key to your account` link
4. Create and paste an ssh key to the textbox. ([How to create an ssh key guide](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent))
5. Click the `Save SSH Key` button
6. Ensure that you are navigated back to hosting configuration screen.

#### Pre-merge Checklist






Related to #69582
